incubator/mission-control/templates/mission-control-certs.yaml

```diff
@@ -0,0 +1,168 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "mission-control.fullname" . }}-certs
+  labels:
+    app: {{ template "mission-control.name" . }}
+    chart: {{ template "mission-control.chart" . }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+data:
+  insight.key: |
```

Comment:
Shouldn't the user be able to bring their own key? Also, keys should probably go into a secret.

Reply:
As of now, this is the only way to provide keys. I tried to convert this to a secret. We are going to have master key concept in this one as well which will eliminate need of passing keys like this.